django/db/backends/sqlite3/creation.py

```diff
@@ -85,6 +97,12 @@ def _clone_test_db(self, suffix, verbosity, keepdb=False):
             except Exception as e:
                 self.log("Got an error cloning the test database: %s" % e)
                 sys.exit(2)
+        # Forking automatically makes a copy of an in-memory database.
+        # Spawn requires migrating to disk which setup_worker_connection will
+        # re-opened in to memory,
```

Comment:
`will re-opened in to ...` should maybe be something like `will re-open in ...`

Reply:
Something like this? 
```
        # Forking automatically makes a copy of an in-memory database.
        # Spawn requires migrating to disk which will be re-opened in 
        # setup_worker_connection.
```

Reply:
Works for me :)